assets/js/modules/analytics/settings/settings-edit.test.js

```diff
@@ -47,10 +47,12 @@ describe( 'SettingsEdit', () => {
 			const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
 			existingTag.accountID = profiles[ 0 ].accountId;
 			existingTag.propertyID = profiles[ 0 ].webPropertyId;
+			const accountID = existingTag.accountID;
+			const propertyID = existingTag.propertyID;
```

Comment:
See other similar comments, I think this would be easier to understand if the assignment happened in the `receive...` calls below. Alternatively, this can be simplified to `const { accountID, propertyID } = existingTag`.

Reply:
Fixed